### PR TITLE
updating README.md information

### PR DESCRIPTION
**Pull request by pangolp**
## Description

- Some links are added.
- PHP 8.x is not supported at the moment.
- Updating from year 2021.
- Discord links and donations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,17 @@
...
 | **Apache Modules** | mod_headers - mod_rewrite - mod_expires |
-| **PHP Extensions** | curl - gd2 - mbstring - mysqli - openssl - soap - gmp|
+| **PHP Extensions** | curl - gd2 - mbstring - mysqli - openssl - soap - gmp |
+
+## Useful Links
+
+* [Website](https://wow-cms.com)
+* [Bugtracker](https://dev.wow-cms.com)
+* [Docs](https://docs.wow-cms.com)
+* [Discord](https://discord.com/invite/fXkHj6GcZm)
```
Comment (DZywolf) [inline README.md:18]:
```suggestion
* [Discord](https://discord.wow-cms.com)
```

File: README.md [hunk 2]
```diff
@@ -6,9 +6,17 @@
...
-| **PHP Version** | **7.1 or newer** is recommended |
+| **PHP Version** | **7.1 or newer** is recommended (Version 8.x is currently not supported.) |
 | **Apache Modules** | mod_headers - mod_rewrite - mod_expires |
-| **PHP Extensions** | curl - gd2 - mbstring - mysqli - openssl - soap - gmp|
+| **PHP Extensions** | curl - gd2 - mbstring - mysqli - openssl - soap - gmp |
+
+## Useful Links
+
+* [Website](https://wow-cms.com)
+* [Bugtracker](https://dev.wow-cms.com)
```
Comment (DZywolf) [inline README.md:16]:
```suggestion

```

**Review by DZywolf — CHANGES_REQUESTED** [review]
This needs to be changed

**Comment by pangolp** [comment]
Suggestions applied Mr. @DZywolf

**Review by DZywolf — APPROVED** [review]

**Pull request merged**